docs/data/data-grid/column-menu/column-menu.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Data Grid - Column menu
+---
+
+# Data grid - Column menu
+
+<p class="description">Customize your columns menu.</p>
+
+## Column menu
+
+Each column header comes with a column menu with quickly accessible grid features like column visibility, sorting, filtering, and others.
+
+It can be accessed by clicking on 3-dots icon that appear on hover on a header cell.
+
+{{"demo": "ColumnMenuGrid.js", "bg": "inline"}}
+
+## Disable column menu
+
+By default, each column header has the column menu enabled. To disable the column menu, set the prop `disableColumnMenu={true}`.
+
+{{"demo": "DisabledColumnMenuGrid.js", "bg": "inline"}}
+
+## Customize column menu
+
+You can customize the column menu either by using `components`, `componentsProps` and `initialItems` supported by column menu component or by using custom component using the `ColumnMenu` slot of the data grid component.
+
+### Add, override, hide and reorder items
```

Comment:
Maybe split this section into four:
- Adding a menu item
- Overriding menu item component
- Hiding a menu item
- Reordering menu items

And show a smaller demo for each one?